manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/tooling/QueryRunner.scala

```diff
@@ -129,9 +132,11 @@ class QueryRunner(formatter: (GraphDatabaseService, Transaction) => InternalExec
       QueryRunResult(queryText, content, formattedResult)
     }
 
-  private def explainSingleQuery(database: RestartableDatabase, queryText: String, placeHolder: ExecutionPlanPlaceHolder) = {
-    val planString = Try(database.execute(s"EXPLAIN $queryText")) match {
+  private def explainSingleQuery(database: RestartableDatabase, queryText: String, placeHolder: QueryResultPlaceHolder, profile: Boolean) = {
```

Comment:
Since this function now does `profile` as well as `explain`, perhaps we should rename it. I'd also prefer to pass the string indicating which of these it is directly instead of a boolean that changes the `explain` to a `profile`.